azure-pipelines.yml

```diff
@@ -40,8 +40,6 @@ jobs:
 
       - script: |
           set -e
-          curl -sSL https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-          dep ensure -v
```

Comment:
This is no longer needed; go will automatically download modules for each submodule during `test` / `vet`, etc.